pkgs/applications/networking/mailreaders/hasmail/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, pkgconfig
+, gobject-introspection
+, pango
+, cairo
+, gtk2
+}:
+
+buildGoModule rec {
+  pname = "hasmail";
+  version = "2019-08-24";
+  name = "${pname}-${version}-unstable";
```

Comment:
I thought we were adding `-unstable` to `pname` now...

Reply:
According to e.g `xsel`, you are right.